### ISSUE TITLE
Ort is failing to build due to error in ort-1.16.0/src/io_binding.rs:56:67

### ISSUE THREAD

**Issue by maxwellflitton** [pykeio/ort#98]
When trying to compile I get the following error:

```
   Compiling ort v1.16.0
error[E0308]: mismatched types
  --> src/io_binding.rs:56:67
   |
56 |             let lengths = unsafe { std::slice::from_raw_parts(lengths_ptr, count).to_vec() };
   |                                    --------------------------              ^^^^^ expected `usize`, found `u64`
   |                                    |
   |                                    arguments to this function are incorrect
   |
note: function defined here
  --> /rustc/8ede3aae28fe6e4d52b38157d7bfe0d3bceef225/library/core/src/slice/raw.rs:92:21
help: you can convert a `u64` to a `usize` and panic if the converted value doesn't fit
   |
56 |             let lengths = unsafe { std::slice::from_raw_parts(lengths_ptr, count.try_into().unwrap()).to_vec() };
   |                                                                                 ++++++++++++++++++++

error[E0277]: a value of type `usize` cannot be made by summing an iterator over elements of type `&u64`
```

**Comment by decahedron1** [comment]
Fixed in [1.16.1](https://crates.io/crates/ort/1.16.1)

**Comment by maxwellflitton** [comment]
@decahedron1 thank you it is working
